File: JavaScript/Faccat/Exercicio11.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('Digite o valor do salário fixo: ', (salarioFixo) => {
  rl.question('Digite o valor recebido por cada carro vendido: ', (comissao) => {
    rl.question('Digite o número de carros vendidos: ', (numeroDeCarrosVendidos) => {
      rl.question('Digite o valor total de vendas: ', (valorDasVendas) => {
        salarioFixo = parseFloat(salarioFixo);
        comissao = parseFloat(comissao);
        numeroDeCarrosVendidos = parseFloat(numeroDeCarrosVendidos);
        valorDasVendas = parseFloat(valorDasVendas);

        const salarioFinal = salarioFixo + (comissao * numeroDeCarrosVendidos) + (valorDasVendas * 5 / 100);
        console.log(`O salário final é: R$ ${salarioFinal.toFixed(2)}`);

        rl.close();
      });
    });
  });
});

File: Java/src/Faccat/Exercicio11.java
package Faccat;

import java.util.Scanner;

public class Exercicio11 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite o valor do salário fixo: ");
        double salarioFixo = scanner.nextDouble();

        System.out.print("Digite o valor recebido por cada carro vendido: ");
        double comissao = scanner.nextDouble();

        System.out.print("Digite o número de carros vendidos: ");
        double numeroDeCarrosVendidos = scanner.nextDouble();

        System.out.print("Digite o valor total de vendas: ");
        double valorDasVendas = scanner.nextDouble();

        double salarioFinal = salarioFixo + (comissao * numeroDeCarrosVendidos) + (valorDasVendas * 0.05);

        System.out.println("O salário final é: R$ " + salarioFinal);

        scanner.close();
    }
}